tests/out/shapes.txt

```diff
@@ -0,0 +1,1774 @@
+shape_id,shape_pt_lat,shape_pt_lon,shape_pt_sequence
+shp_19914,53.0507001,8.7896481,0
```

Comment:
@mehdisadeghi I guess that you decided to use the "shp" prefix to differentiate the use of the same relation id for several output files. I think this can lead to some problems down the road since the use of that prefix means that we need to treat those fields as strings not as integers anymore which greatly impact the memory consumption. I would propose the following alternatives:
- use the same id for shape_id, trip_id, etc. Since those fields are tagged directly in the files, there shouldnt be any problem in using the same id for several fields as long as they have a different tag.
- use an increment counter for each id
- use a concatenation of relation id and changeset number. Both of those fields are integers so concatenating the two values should also produce an integer value.

what do you think?

Reply:
@carocad Yes this can definitely be improved. The following points come to my mind:

 - GTFS description asks only for _dataset unique_ ids, not integers.
 - Any solution that we come up with should reproduce the same ids over different runs
 - concatenating ids and changesets sounds like a good idea!

Reply:
@carocad I used relation id as shape id.

Reply:
@carocad btw, we shouldn't consider every gtfs file to have integers as IDs, because they don't.